woocommerce/woocommerce-android
# Variation description

This PR adds variation description editing capability.

![image](https://user-images.githubusercontent.com/1522856/88943578-f28f8300-d28b-11ea-95a2-b20afc804355.png)

**To test:**

0. Go to variation details screen
1. **Notice there is an arrow in the description property**
2. Tap on a description property
3. **Notice an Aztec editor is opened with the description**
4. Change the description
5. Tap on Done
6. **Notice the variation details screen is shown with the description updated**
7. **Notice the Update button appears**
8. Tap on the Update button
9. Leave the variation details
10. Open the same variation again
11. **Notice the description was persisted**
